examples/duoswap-interactive/index.rsh

```diff
@@ -0,0 +1,296 @@
+'reach 0.1';
```

Comment:
I don't like that this file is duplicated. I think that you should just make an interactive version in the other directory